datasets/bbga/bbga.json

```diff
@@ -1,6 +1,7 @@
 {
   "type": "dataset",
   "id": "bbga",
+    "path": "bbga",
```

Comment:
The indentation is inconsistent. jq+sponge (from moreutils) can fix this, if your editor doesn't:

```sh
for f in datasets/*/*.json; do
    jq . "$f" | sponge "$f"
done
```

Reply:
I assumed the precommit hooks would fix it. Weird, they fixed all kinds of formatting and white space stuff, but missed this.